### ISSUE TITLE
Cannot download export template from macOS editor

### ISSUE THREAD

**Issue by BraveEvidence** [godotengine/godot#95755]
### Tested versions

Godot 4.3 stable

### System information

Godot v4.3.stable - macOS 14.6.1 - Vulkan (Mobile) - integrated Apple M1 - Apple M1 (8 Threads)

### Issue description

I am trying to download the android build template for Godot but get an error saying Can't connect to the mirror

### Steps to reproduce

Create a Godot Project. Click Project => Install Android Build Template => Click Manage Templates => Click Download & Install

### Minimal reproduction project (MRP)

https://github.com/user-attachments/assets/f4eea114-d009-4d11-98b4-1348cb844ddf

**Comment by DistroRickDev** [comment]
Have you tried to "Open in Web browser" to check if the template file downloads?

**Comment by BraveEvidence** [comment]
Cool that worked but I will keep this open for now till the other button's functionality is fixed

**Comment by BraveEvidence** [comment]
ok I had to unzip the folder which I downloaded from "Open in Web browser", delete the build folder from the android folder of my Godot project and then click on install again Project => Install Android Build Template

**Comment by DistroRickDev** [comment]
I've tested myself under:
Godot v4.3.stable - Linux (Ubuntu 22.04) - Vulkan (Forward+)

And I was able to obtain it over the Click Download & Install button, from the error it seemed like a network issue on https_download_complete:
https://github.com/godotengine/godot/blob/1bd740d18d714f815486b04bf4c6154ef6c355d9/editor/export/export_template_manager.cpp#L183

As it seems to download the template the engine makes use of the editor https host and port configurations:
https://github.com/godotengine/godot/blob/1bd740d18d714f815486b04bf4c6154ef6c355d9/editor/export/export_template_manager.cpp#L158C28-L158C38

Do you have any proxy specific proxy configuration in the editor?

**Comment by BraveEvidence** [comment]
No, did not changed anything in the editor. Just downloaded Godot 4.3 and the first thing which I did was to download the template

**Comment by Calinou** [comment]
> Perhaps “Resume Download” would be quite good so that the download does not have to be constantly restarted.

HTTPRequest does not support range requests, so that would need to be implemented first.

**Comment by Alex2782** [comment]
https://godotengine.org/mirrorlist/4.3.stable.json

`best available` is always Github I think. 
@BraveEvidence Can you continue to reproduce this with `Github` and `TuxFamily`?

`Menu -> Editor -> Export Template Manager`

![image](https://github.com/user-attachments/assets/8d26e92f-620c-4ad2-a555-c96ba97d4ae6)

https://github.com/godotengine/godot/blob/e3550cb20f5d6a61befaafb7d9cbdb57b24870e4/editor/export/export_template_manager.cpp#L581-L584

**Comment by myselfuser1** [comment]
@Alex2782  Yes i can reproduce it using Github & TuxFamily. Check [this](https://private-user-images.githubusercontent.com/24708929/358913979-f4eea114-d009-4d11-98b4-1348cb844ddf.mp4?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ1NTQ2MDMsIm5iZiI6MTcyNDU1NDMwMywicGF0aCI6Ii8yNDcwODkyOS8zNTg5MTM5NzktZjRlZWExMTQtZDAwOS00ZDExLTk4YjQtMTM0OGNiODQ0ZGRmLm1wND9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI1VDAyNTE0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiNTc5MWMwN2EzNmFlZjI0MTg2OGM2MDczODYyYmMyNmE0Y2MxZDRlY2ZjMTU5MzliNmI4MzNmMjRhYjhhMmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZNsQE5oEbdHwCA3ZinaigFW3UcQhdwDJNu9DkAjFdus)

**Comment by honey2006** [comment]
don't download from github one, use the other one!

**Comment by AThousandShips** [comment]
> don't download from github one, use the other one!

Why?

**Comment by CreatedBySeb** [comment]
@BraveEvidence I see you have Sophos Antivirus based on your Menu Bar, can you reproduce this issue if you disable Sophos and try to download and install the export templates? I am wondering if it is possibly blocking the GitHub releases download by Godot because it thinks it is suspicious in some way, since I am able to download the export templates fine on macOS Sequoia with 4.3.stable.
